src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -1039,7 +1039,7 @@ N2NTMToadlet.tooLarge=The file you have uploaded through your browser is ${attem
 N2NTMToadlet.sizeWarning=Please note that browser-based uploading has a limit of ${limit}.
 N2NTMToadlet.uploadFailed=Failed to upload file.
 N2NTMUserAlert.delete=Delete
-N2NTMUserAlert.header=From: ${from} (composed ${composed} | sent ${sent} | received ${received})
+N2NTMUserAlert.header=From: ${from} (sent ${sent})
```

Comment:
My apologies for not catching this earlier, but `sentTime` can apparently be deceiving: it is set as the time at which the N2N message is dispatched to the Node's MessageCore ([source][0]) where it is queued for transmission over the network, if and only if the peer is currently connected. It should be really close to `receivedTime` if we ignore clock skew problems.

`composedTime` might be the one you are after knowing.

[0]: https://github.com/freenet/fred/blob/next/src/freenet/node/DarknetPeerNode.java#L610